### DISCUSSION TITLE
routeOverride is deprecated and  is slated for removal.

### DISCUSSION THREAD

**Discussion by abd3lraouf** [varabyte/kobweb#477] [Q&A] [answered]
Previously I was setting the routeOverride globally to `KebabCase`

```kotlin
    markdown {
        routeOverride.set(KebabCase) // e.g. "ExamplePage" to "example-page"
    }
```

I've also added some custom tasks that will traverse the article list to retrieve the route override and apply it to the article path.

```kotlin
    val fileOverride = kobweb.markdown.routeOverride.get()
    ..... 
    val articleRoute = fileOverride(blogArticle.nameWithoutExtension)
```

Now It's deprecated, and when I remove it completely article paths are in this format `ArticlePath` and not in `article-path`, so I thought `KebabCase` was already set to default

The reason I'm taking this path is that I don't want to define an override for each article I have manually.

**Comment by bitspittle** [comment, marked as answer]
The default now is kebab case. See also: https://github.com/varabyte/kobweb#legacy-routes

If you changed your route override logic to output a 💩 emoji you'll see the final result will still be kebab case

**Reply by abd3lraouf** [reply]
Thank you
